ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=18:ompthreads=1:ncpus=18:mem=80GB
+#PBS -l select=1:ncpus=18:mem=80GB
```

Comment:
Please use developer provided resource request.  Undo this change.

Reply:
Change undo is done.